source/_components/mysensors.markdown

```diff
@@ -13,11 +13,11 @@ featured: true
 ha_iot_class: "Local Push"
 ---
 
-The [MySensors](https://www.mysensors.org) project combines Arduino boards with NRF24L01 radio boards to build sensor networks. The component will automatically add all available devices to Home Assistant, after [presentation](#presentation) is done.
+The [MySensors](https://www.mysensors.org) project combines devices like Arduino, ESP8266, Raspberry Pi, NRF24L01+ and RFM69 to build affordable sensor networks. This component will automatically add all available devices to Home Assistant, after [presentation](#presentation) is done. That is, you do not need to add config entries for the devices for them to be added. Go to the **states** section of the developer tools to find the devices that have been identified.
```

Comment:
"config entries" may be confused for config entries as in https://developers.home-assistant.io/docs/en/config_entries_index.html. Can we call it something else eg "anything in the config"?